deps/triemap/triemap.c

```diff
@@ -190,14 +190,14 @@ int TrieMapNode_Add(TrieMapNode **np, char *str, tm_len_t len, void *value, Trie
     __trieMapNode_children(n)[char_offset] = child;
     return rv;
   }
-  
-  ptr = childKeys; 
+
+  ptr = childKeys;
   while(ptr < childKeys + n->numChildren && *ptr < c) {++ptr;}
   *np = __trieMapNode_AddChildIdx(n, str, offset, len, value, ptr - childKeys);
   return ++rv;
 }
 
-int TrieMap_Add(TrieMap *t, char *str, tm_len_t len, void *value, TrieMapReplaceFunc cb) {
+int TrieMap_Add(TrieMap *t, const char *str, tm_len_t len, void *value, TrieMapReplaceFunc cb) {
```

Comment:
Excellent work! I had meant to point this out as a feature for a future PR. But getting it done here is even better.